docs/content/querying/filters.md

```diff
@@ -81,35 +81,6 @@ The following matches any dimension values for the dimension `name` between `'ba
 }
 ```
 
-### Extraction filter
```

Comment:
@gianm put the extraction filter docs back, marked it deprecated with a "note caution" block